tests/conftest.py

```diff
@@ -166,3 +166,14 @@ def manager_in_group(user_factory, research_group_factory):
     group, memberships = research_group_factory(number_of_members=0)
     GroupMembership.objects.create(group=group, member=manager, is_manager=True)
     return manager, group
+
+
+@pytest.fixture
+def no_manager_group(user_factory, research_group_factory):
+    """Return a user who is not a manager in a research group."""
+    from imperial_coldfront_plugin.models import GroupMembership
+
+    manager = user_factory()
+    group, memberships = research_group_factory(number_of_members=0)
+    GroupMembership.objects.create(group=group, member=manager, is_manager=False)
+    return manager, group
```

Comment:
This fixture seems to be equivalent to just calling `research_group_factory(number_of_members=1)` so I don't think we need a dedicated fixture for this.